cli/azd/pkg/devcenter/config.go

```diff
@@ -0,0 +1,56 @@
+package devcenter
+
+import (
+	"fmt"
+
+	"github.com/azure/azure-dev/cli/azd/pkg/project"
+)
+
+const (
+	// Environment variable names
+	DevCenterNameEnvName          = "AZURE_DEVCENTER_NAME"
+	DevCenterCatalogEnvName       = "AZURE_DEVCENTER_CATALOG"
+	DevCenterProjectEnvName       = "AZURE_DEVCENTER_PROJECT"
+	DevCenterEnvTypeEnvName       = "AZURE_DEVCENTER_ENVIRONMENT_TYPE"
+	DevCenterEnvDefinitionEnvName = "AZURE_DEVCENTER_ENVIRONMENT_DEFINITION"
+	DevCenterEnvUser              = "AZURE_DEVCENTER_ENVIRONMENT_USER"
+
+	PlatformKindDevCenter project.PlatformKind = "devcenter"
+)
+
+var (
+	// Environment configuration paths
+	DevCenterNamePath          = fmt.Sprintf("%s.name", ConfigPath)
+	DevCenterCatalogPath       = fmt.Sprintf("%s.catalog", ConfigPath)
+	DevCenterProjectPath       = fmt.Sprintf("%s.project", ConfigPath)
+	DevCenterEnvTypePath       = fmt.Sprintf("%s.environmentType", ConfigPath)
+	DevCenterEnvDefinitionPath = fmt.Sprintf("%s.environmentDefinition", ConfigPath)
+	DevCenterUserPath          = fmt.Sprintf("%s.user", ConfigPath)
```

Comment:
You can make all of these to be `constants` as well, like

```
const (
  DevCenterNamePath          = ConfigPath + ".name"
)
```

Is there a reason why we might want not to assign the value until runtime?